docs/FAQ.md

```diff
@@ -37,6 +37,23 @@ See [revsets] and [templates] for further guidance.
 
 Use `jj log -r ..`. The `..` [operator] lists all visible commits in the repo, excluding the root (which is never interesting and is shared by all repos).
 
+### Can I monitor how `jj log` evolves?
+
+The simplest way to monitor how the history as shown by `jj log` evolves is by using the `watch(1)` command (or [hwatch](https://github.com/blacknon/hwatch) or [viddy](https://github.com/sachaos/viddy)).
```

Comment:
Optional: I always give a link to https://github.com/blacknon/hwatch?tab=readme-ov-file#configuration when mentioning hwatch, since its defaults are quite weird.

You could also link to https://man7.org/linux/man-pages/man1/watch.1.html and/or https://gitlab.com/procps-ng/procps for `watch`.

Reply:
I don't know the tool. Make a call, do you want the change or not?

Reply:
Had another edit to make, so I added the links now.